### ISSUE TITLE
canFloat.defaultPosition ignored if initial widget state is WidgetState.Hidden

### ISSUE THREAD

**Issue by arnoldas-v** [iTwin/appui#551]
**Describe the bug**
`canFloat.defaultPosition` and `canFloat.defaultSize` ignored if initial default widget state is `WidgetState.Hidden` and later set to `WidgetState.Floating`.
Widget always appears at constant small offset from top-left corner.

**To Reproduce**

Example widgetConfig:
```
{
    id: "wid", defaultState: WidgetState.Hidden, content: <div></div>,
    canFloat: { isResizable: true, defaultPosition: { x: 500, y: 500 }, defaultSize: { width: 500, height: 500 } }
}
```
Later make the widget floating like so:
`UiFramework.frontstages.activeFrontstageDef?.findWidgetDef("wid")?.setWidgetState(WidgetState.Floating);`

**Expected behavior**
Widget initially opens at specified location and size.

**Desktop (please complete the applicable information):**

- OS: Windows11
- Browser: Electron 4.1.9
- iTwin.js AppUI Version 4.6.1
- iTwin.js Version 4.1.9

**Comment by raplemie** [comment]
Hi @arnoldas-v, a fix for this was released today in AppUI 4.6.4, let us know if this fixes your issue!

**Comment by GintV** [comment]
@raplemie Seems to be fixed on 4.6.4, thanks!
